src/react/components/CampusSolutionsLink/CampusSolutionsLinkContainer.js

```diff
@@ -2,78 +2,85 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import _ from 'lodash';
 
-import { fixLastQuestionMark, updateQueryStringParameter } from '../../helpers/links';
+import {
+  fixLastQuestionMark,
+  updateQueryStringParameter,
+} from '../../helpers/links';
 import CampusSolutionsLink from './CampusSolutionsLink';
 import CampusSolutionsOutboundLink from './CampusSolutionsOutboundLink';
 
 const propTypes = {
   linkObj: PropTypes.object.isRequired,
-  children: PropTypes.any.isRequired
+  children: PropTypes.any.isRequired,
 };
 
-class CampusSolutionsLinkContainer extends React.Component {
-  constructor(props) {
-    super(props);
-    this.decorateLinkUrl = this.decorateLinkUrl.bind(this);
-    this.getLinkConfig = this.getLinkConfig.bind(this);
-    this.getUcFromParamConfig = this.getUcFromParamConfig.bind(this);
-  }
-  decorateLinkUrl(linkUrl, includeUcFrom, includeUcFromLink, includeUcFromText, ccCacheString, ccPageName, ccPageUrl) {
-    if (/^http/.test(linkUrl) && includeUcFrom === true) {
-      linkUrl = fixLastQuestionMark(linkUrl);
+const CampusSolutionsLinkContainer = () => {
+  return null;
+};
```

Comment:
@developish - What's happening here? Why is CampusSolutionsLinkContainer being defined as something that just returns null, with everything else commented out?

Reply:
Another in-progress thing that I ended up not using